src/pages/LandingPage.js

```diff
@@ -133,13 +135,7 @@ export const LandingPage = () => {
       ),
     // width is set to prevent text from overlapping the background image and decreasing legibility
     div({ style: { maxWidth: 'calc(100% - 460px)' } }, makeDocLinks(getEnabledBrand().docLinks)),
-    div({ style: { display: 'flex', margin: '2rem 0 1rem 0' } }, [
-      makeCard('workspaces', 'View Workspaces', [
-        'Workspaces connect your data to popular analysis tools powered by the cloud. Use Workspaces to share data, code, and results easily and securely.',
-      ]),
-      makeCard('library-showcase', 'View Examples', 'Browse our gallery of showcase Workspaces to see how science gets done.'),
-      makeCard('library-datasets', 'Browse Data', 'Access data from a rich ecosystem of data portals.'),
-    ]),
+    div({ style: { display: 'flex', margin: '2rem 0 1rem 0' } }, makeCard(getEnabledBrand().landingPageCards || landingPageCardsDefault)),
```

Comment:
The children argument to hyperscript helpers functions should always be an array.

```suggestion
    div({ style: { display: 'flex', margin: '2rem 0 1rem 0' } }, [makeCard(getEnabledBrand().landingPageCards || landingPageCardsDefault)]),
```

Reply:
Also, we can confine this logic to the brands module by setting every other brand's `landingPageCards` option to the default value.

```suggestion
    div({ style: { display: 'flex', margin: '2rem 0 1rem 0' } }, makeCard(getEnabledBrand().landingPageCards),
```